docs/modules/ROOT/nav-how-to-guides.adoc

```diff
@@ -13,6 +15,6 @@
 ** xref:how-to-guides/delete_application.adoc[Web UI: Deleting an application].
 
 ////
-I'm commenting out this xref for now because Burr said this page is currently unsupported. --Christian (csears@redhat.com), 2/16/2023
+Commenting out this xref for now because Burr said this page is currently unsupported. --Christian (csears@redhat.com), 2/16/2023
 ** xref:cli/proc_release_application.adoc[Releasing an application]
```

Comment:
We haven't built UI for the release service yet. It'll be necessary for our internal customers, but not something that external customers would interact with yet. After Summit, we'll start working on that.